source/helpers/abbreviate-string.ts

```diff
@@ -0,0 +1,3 @@
+export default function abbreviateString(string_: string, length: number, abbrevMarker = '…'): string {
+	return string_.slice(0, length) + abbrevMarker;
```

Comment:
This is not a condition, the result is unchanged. I meant:

```suggestion
	return string.length < length ? string : string_.slice(0, length) + '…';
```

Reply:
Sorry my bad 🥲